string  command
int32   step_num
float64 step_time
float64 step_length
float64 side_step_length
float64 step_angle_rad